html5lib/html5parser.py

```diff
@@ -204,6 +191,9 @@ def mainLoop(self):
         DoctypeToken = tokenTypes["Doctype"]
         ParseErrorToken = tokenTypes["ParseError"]
 
+        type_names = {value: key for key, value in tokenTypes.items()}
```

Comment:
I suppose one could skip computing this unless `debug` is set, but probably not worthwhile.